cfn-resources/federated-database-instance/cmd/resource/resource.go

```diff
@@ -28,13 +28,12 @@ import (
 	"github.com/mongodb/mongodbatlas-cloudformation-resources/util/logger"
 	progress_events "github.com/mongodb/mongodbatlas-cloudformation-resources/util/progressevent"
 	"github.com/mongodb/mongodbatlas-cloudformation-resources/util/validator"
-	"github.com/spf13/cast"
-	atlasSDK "go.mongodb.org/atlas-sdk/v20231115002/admin"
+	atlasSDK "go.mongodb.org/atlas-sdk/v20231115008/admin"
```

Comment:
if using latest SDK please remove package alias atlasSDK so you can use just admin. CreateFederatedDatabaseApiParams, etc.

Reply:
Done!